browser/js/adminPanel/admin.js

```diff
@@ -41,7 +54,13 @@ app.config(function ($stateProvider) {
         },
         resolve: {
           topics: function(TopicFactory) {
-            return TopicFactory.fetchAll();
+            return TopicFactory.fetchAll()
```

Comment:
This is inefficient, potentially hundreds of database calls (1 for each topic in the database).  Let's create a route that gets all prereq/subseq for all topics (maybe /api/topics/all) or change the existing route to optionally take an options parameters, and if present adds the prereqs/subseq to the response.